### DISCUSSION TITLE
Possible update to Pull Request Template

### DISCUSSION THREAD

**Discussion by at-the-vr** [withastro/starlight#1084] [Feature Requests]
### What version of `starlight` are you using?

v0.12.1

### What is your idea?

- Comment certain portion of [Starlight Repo's PR Template](https://github.com/withastro/starlight/blob/main/.github/PULL_REQUEST_TEMPLATE.md)
- Make it similar to [Astro Docs Repo PR Template](https://github.com/withastro/docs/blob/main/.github/PULL_REQUEST_TEMPLATE.md)

### Why is this feature necessary?

- Lesser Text to comment/think about during PRs

### Do you have examples of this feature in other projects?

Majorly it includes commenting the bullet points so that they are not replicated in the PR once published
But I am open to any suggestion as well 😄 

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Thanks for the suggestion. Personally, I think we could straight up remove the first “What kind of changes does this PR include?” section. I don’t find it helps me much when reviewing and we already automatically label things for docs/core/i18n changes.

Curious what others think though!

**Reply by HiDeoo** [reply]
> I don’t find it helps me much when reviewing

Same for me, I mostly focus on the _Description_ section as I feel tags and the files list covers the _What kind of changes does this PR include?_ one.

**Reply by at-the-vr** [reply]
Great, if I use [Astro Docs PR Template](https://github.com/withastro/docs/blob/main/.github/PULL_REQUEST_TEMPLATE.md) minus the suggested label text, does that concludes this discussion?

**Reply by delucis** [reply]
I'd say keep the current Starlight template and remove that section — I quite like our questions and don't find the "suggested labels" question in the docs template so helpful in Starlight's case.

**Comment by at-the-vr** [comment]
Marking this Discussion closed, GitHub apparently did not

**Reply by delucis** [reply]
Thanks! Yeah, GitHub doesn’t have a way to auto-close discussions linked to PRs unfortunately. One drawback of using them for feature requests, although I still like the separation of issues = bugs and discussions = ideas.
